components/Card/card.module.scss

```diff
@@ -0,0 +1,10 @@
+.card {
+  padding: 16px;
+}
+
+.cardContent {
+  padding: 0;
+  &:last-child {
+    padding: 0;
```

Comment:
adding `&:last-child` here with same value make sense only in case if your UI library has own `:last-child` like
```
&:last-child {
    padding: 15px;
}
```
is it true?

Reply:
yes, it is. I overridden styles